docs/NGramTransform-sparkcompute.md

```diff
@@ -15,8 +15,6 @@ Properties
 
 **ngramSize:** NGram size. (Macro-enabled)
 
-**outputField:** Transformed field for sequence of n-gram.
```

Comment:
why is this being removed?

Reply:
I made the schema constant with a constant field name, like in a lot of other plugins we do. Since nothing else can really be in schema in this plugin.

Reply:
This just means we are hardcoding the output field to `ngrams`

Reply:
Done